testing/InstallTestbed.ps1

```diff
@@ -0,0 +1,338 @@
+param (
+    [Parameter(Mandatory = $true)]
+    [string]$ResourceGroup,
+
+    [string]$VMName = "DC1",
+    [string]$LinuxVMName = "LS1",
+    [int]$NumberOfClients = 2
```

Comment:
$NumberOfClients defaults to 2 here but defaults to 1 on SetupTestbed.ps1 which I could see causing friction for any users running these two scripts without explicitly specifying the number of clients. I don't have a real preference for 1 or 2 but I think the defaults for both scripts should match.

Reply:
Done. I set them to 2 so it is easier to run for testing.